docs/misc/Acrylic config.fr.md

```diff
@@ -0,0 +1,47 @@
+# Guide de configuration d'Acrylic
+Une autre manière que d'adapter son fichier host existe pour résoudre les noms de domaine en `.test`.
+Cette solution est appelée Acrylic qui est un résolveur de DNS local et qui a pour avantage de pouvoir rediriger d'un seul coup toutes les adresses en `.test` vers votre machine virtuelle.
+## Installation d'Acrylic
+Télécharger [Acrylic](https://mayakron.altervista.org/support/acrylic/Home.htm) setup for windows. Le logiciel étant un peu ancien, celui-ci peut déclencher des alertes de sécurité de la part de votre antivirus. Exécutez le programme en tant qu'administrateur afin de l'installer.
+
+## Configurer Acrylic
+Dans le menu démarer, chercher "Acrylic" et cliquer dans le panneau de droite sur "Ouvrir l'emplacement du fichier".
+![acrylic_open_path.png](../captures/acrylic_open_path.png)
+Dans l'explorateur qui s'est ouvert, ouvrir Acrylic UI avec vos droits administrateurs.
```

Comment:
Leave blank lines around the screen captures, otherwise the text is on the same line